src/Domain/DonationService.php

```diff
@@ -711,8 +711,11 @@ public function findAllCompleteForCustomerAsAPIModels(StripeCustomerId $stripeCu
     /**
      * @throws PaymentIntentNotSucceeded
      * */
-    public function confirmDonationWithSavedPaymentMethod(Donation $donation, StripePaymentMethodId $paymentMethodId): void
-    {
+    public function confirmDonationWithSavedPaymentMethod(
+        Donation $donation,
+        StripePaymentMethodId $paymentMethodId,
+        bool $offSession = false,
```

Comment:
not a blocker but maybe worth removing the default value here so we can't forget to set it if we add any more calls to this function. There are only a few calls to it at the moment.

Reply:
Done in 6a67ac1ce30c6d3dd543ff086341e5d03f202c36